skipchain/skipchain.go

```diff
@@ -211,6 +212,7 @@ func (s *Service) StoreSkipBlockInternal(psbd *StoreSkipBlock) (*StoreSkipBlockR
 		random.Bytes(bl[:], random.New())
 		prop.BackLinkIDs = []SkipBlockID{SkipBlockID(bl[:])}
 		prop.GenesisID = nil
+		prop.SignatureScheme = BdnSignatureSchemeIndex
```

Comment:
A comment explaining the policy (starting with release v3.1.0, new skipchains default to BDN because...) would be helpful here.

And also perhaps someplace in the top README.md, if there's a place that talks about interoperability guarantees?